sui/operators.js

```diff
@@ -0,0 +1,297 @@
+
+    const tx = new Transaction();
+
+    const [cap, loanedCap] = tx.moveCall({
+        target: `${contractConfig.address}::operators::loan_cap`,
+        arguments: [tx.object(operatorId), tx.object(operatorCapId), tx.pure(bcs.Address.serialize(gasCollectorCapId).toBytes())],
+        typeArguments: [`${gasServiceConfig.address}::gas_service::GasCollectorCap`],
+    });
+
+    tx.moveCall({
+        target: `${gasServiceConfig.address}::gas_service::collect_gas`,
+        arguments: [tx.object(gasServiceConfig.objects.GasService), cap, tx.pure.address(receiver), tx.pure.u64(amount)],
+    });
+
+    tx.moveCall({
+        target: `${contractConfig.address}::operators::restore_cap`,
+        arguments: [
+            tx.object(operatorId),
+            tx.object(operatorCapId),
+            tx.pure(bcs.Address.serialize(gasCollectorCapId).toBytes()),
+            cap,
+            loanedCap,
+        ],
+        typeArguments: [`${gasServiceConfig.address}::gas_service::GasCollectorCap`],
+    });
+
+    const receipt = await broadcast(client, keypair, tx);
+
+    printInfo('Gas collected', receipt.digest);
+}
+
+async function refund(keypair, client, gasServiceConfig, contractConfig, args, options) {
+    const [messageId] = args;
+    const amount = options.amount;
+    const receiver = options.receiver || keypair.toSuiAddress();
+
+    if (!gasServiceConfig) {
+        throw new Error('Gas service package not found.');
+    }
+
+    if (!contractConfig) {
+        throw new Error('Operators package not found.');
+    }
+
+    const operatorId = contractConfig.objects.Operators;
+    const gasCollectorCapId = await getGasCollectorCapId(client, gasServiceConfig, contractConfig);
+    const operatorCapId = await findOwnedObjectId(client, keypair.toSuiAddress(), `${contractConfig.address}::operators::OperatorCap`);
+
+    const tx = new Transaction();
+
+    const [cap, loanedCap] = tx.moveCall({
+        target: `${contractConfig.address}::operators::loan_cap`,
+        arguments: [tx.object(operatorId), tx.object(operatorCapId), tx.pure(bcs.Address.serialize(gasCollectorCapId).toBytes())],
+        typeArguments: [`${gasServiceConfig.address}::gas_service::GasCollectorCap`],
+    });
+
+    tx.moveCall({
+        target: `${gasServiceConfig.address}::gas_service::refund`,
+        arguments: [
+            tx.object(gasServiceConfig.objects.GasService),
+            cap,
+            tx.pure.string(messageId),
+            tx.pure.address(receiver),
+            tx.pure.u64(amount),
+        ],
+    });
+
+    tx.moveCall({
+        target: `${contractConfig.address}::operators::restore_cap`,
+        arguments: [
+            tx.object(operatorId),
+            tx.object(operatorCapId),
+            tx.pure(bcs.Address.serialize(gasCollectorCapId).toBytes()),
+            cap,
+            loanedCap,
+        ],
+        typeArguments: [`${gasServiceConfig.address}::gas_service::GasCollectorCap`],
+    });
```

Comment:
```suggestion
    operatorMoveCall(
        tx,
        `${gasServiceConfig.address}::gas_service::GasCollectorCap`,
        gasCollectorCapId,
        (cap) => {
            target: `${gasServiceConfig.address}::gas_service::refund`,
            arguments: [
                tx.object(gasServiceConfig.objects.GasService),
                cap,
                tx.pure.string(messageId),
                tx.pure.address(receiver),
                tx.pure.u64(amount),
            ],
        });
```
Perhaps create a helper method, say `operatorMoveCall(tx, capTypeName, capId, callWithLoanedCap)` to generalize executing arbitrary move call using operators contract.